File: businesslib/src/main/java/com/wujia/businesslib/base/NetConfigWrapper.java
package com.wujia.businesslib.base;

import com.wujia.businesslib.Constants;
import com.wujia.lib_common.data.network.NetConfig;

import okhttp3.Interceptor;


public class NetConfigWrapper {

    public static NetConfig create() {

        return create(Constants.BASE_URL);
    }

    public static NetConfig create(String baseUrl) {
//        NetConfig.debug = BuildConfig.DEBUG;
        NetConfig.debug = true;
        NetConfig config = new NetConfig();
        config.configInterceptors(new Interceptor[]{ new TokenInterceptor()});
        config.baseURL = baseUrl;
        return config;
    }

}

File: businesslib/src/main/java/com/wujia/businesslib/base/NetConfigWrapper.kt
package com.wujia.businesslib.base

import com.wujia.lib_common.base.Constants
import com.wujia.lib_common.data.network.NetConfig

import okhttp3.Interceptor


object NetConfigWrapper {

    @JvmOverloads
    fun create(baseUrl: String = Constants.BASE_URL): NetConfig {
        //        NetConfig.debug = BuildConfig.DEBUG;
        NetConfig.debug = true
        val config = NetConfig()
        config.configInterceptors(arrayOf<Interceptor>(TokenInterceptor()))
        config.baseURL = baseUrl
        return config
    }

}
